avatar/avatar.html

```diff
@@ -0,0 +1,94 @@
+            </li>
+
+            <li class="inline-item">
+                <a class="links" target="_blank" href="https://twitter.com/ThoramVeekshith"><img
+                        src="/avatar/images/icons8-twitter-50.png" id="tweet-img" alt="twitter"></a>
+            </li>
+
+            <li class="inline-item">
+                <a class="links" target="_blank" href="https://www.linkedin.com/in/veekshith-thoram-72b973217/"><img
+                        src="/avatar/images/icons8-linkedin-30.png" id="linkedin-img" alt="linkedin"></a>
+            </li>
+        </ul>
+
+    </nav>
+    <div class="container">
+        <div class="sidenav-container">
+            <nav class="side-nav">
+                <ul class="list-non-bullet">
+                    <li class="side-nav-link"><a class="links" href="#">Avatar</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Alert</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Badge</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Button</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Card</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Image</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Input</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Lists</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Toast</a></li>
+                    <li class="side-nav-link"><a class="links" href="#">Text</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="main-container">
+            <div class="avatar-container">
+                <h1>Avatar</h1>
+                <div class="desc-txt">An Avatar is a manifestation of an image or a person or an idea.</div>
+
+                <div class="line"></div>
+
+                <h2>Avatar with Image</h2>
+
+                <li class="avatar-li">
+                    <ul class="avatar-ul"><img src="/avatar/images/spud icon.jpg" id="avatar-lg" alt="avatar"></ul>
+                    <ul class="avatar-ul"><img src="/avatar/images/spud icon.jpg" id="avatar-md" alt="avatar"></ul>
+                    <ul class="avatar-ul"><img src="/avatar/images/spud icon.jpg" id="avatar-sm" alt="avatar"></ul>
+                </li>
+
+                <div class="desc-txt">
+                    By using the below code snippet you can create avatar with image on it of your choice just by
+                    changing the src in the img tag.
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/622efdececc7d70015fadfe2"
+                    style="width: 70%; height: 125px;" frameborder="0"></iframe>
+
+                <div class="line"></div>
+
+                <h2>Avatar with Text</h2>
+
+
+                <div class="text-container">
```

Comment:
Here also indentation and spacing is inconsistent, use a code formatter. I use prettier that might help.